app/models/assignment.rb

```diff
@@ -4,4 +4,6 @@ class Assignment < ActiveRecord::Base
 
   belongs_to :week
   belongs_to :employee
+
+  scope :upcoming, -> { where('week_id > ?', Week.upcoming.first.id) }
```

Comment:
Should this be `>=`? Probably, although I imagine managers and employees would figure it out in either case.